textproto/header.go

```diff
@@ -309,6 +309,90 @@ func isSpace(c byte) bool {
 	return c == ' ' || c == '\t'
 }
 
+func validHeaderFieldByte(b byte) bool {
+	return int(b) < len(isTokenTable) && isTokenTable[b]
+}
+
+var isTokenTable = [127]bool{
```

Comment:
Instead of this big table, can we use the check described in the RFC:

```go
c >= 33 && c <= 126 && c != ':'
```

Reply:
Sure. This code was copied from the Go textproto library. Not sure why they use a symbol table